bindings/python/build_tools/cmake/iree_py_extension.cmake

```diff
@@ -212,22 +212,29 @@ function(iree_complete_py_extension_link_options)
         # For MSVC, also add a `-WHOLEARCHIVE:` version of the dep.
         # CMake treats -WHOLEARCHIVE[:lib] as a link flag and will not actually
         # try to link the library in, so we need the flag *and* the dependency.
+        # For macOS, also add a `-Wl,-force_load` version of the dep.
         if(MSVC)
           get_target_property(_ALIASED_TARGET ${_DEP} ALIASED_TARGET)
           if (_ALIASED_TARGET)
             list(APPEND _ALWAYS_LINK_DEPS "-WHOLEARCHIVE:${_ALIASED_TARGET}")
           else()
             list(APPEND _ALWAYS_LINK_DEPS "-WHOLEARCHIVE:${_DEP}")
           endif()
+        elseif("${CMAKE_SYSTEM_NAME}" STREQUAL "Darwin")
```

Comment:
Is there a reason you chose this versus elseif(APPLE) ?

Reply:
I just wasn't aware of this build variable :) Changed my implementation, thanks for pointing out!